components/top-nav/src/atoms/logo-anchor/index.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'react-emotion';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { WHITE, YELLOW, BLACK } from 'govuk-colours';
+
+const LogoAnchor = ({ anchorType, ...props }) => {
+  const StyledLogoAnchor = styled(anchorType)({
```

Comment:
As noted against the `asAnchor` HOC source above though, it'd be good to see the style duplication removed from here